web/src/components/Prices/HollaexToken.js

```diff
@@ -0,0 +1,570 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
There are several `let` variables that can be defined as a `const`.

Reply:
Updated possible place